src/pocketmine/network/mcpe/protocol/StartGamePacket.php

```diff
@@ -98,7 +98,8 @@ class StartGamePacket extends DataPacket{
 	public $isTexturePacksRequired = true;
 	/** @var array */
 	public $gameRules = [ //TODO: implement this
-		"naturalregeneration" => [1, false] //Hack for client side regeneration
+		"naturalregeneration" => [1, false], //Hack for client side regeneration
+		"experimentalgameplay" => [1, true] // Enables client-side scripts
```

Comment:
It might be an idea to have a configuration option for this. As the name suggestions, it is **experimental** and shouldnt be enabled by default in case Mojang manage to break something later on

Reply:
Yeah, it'd be a good idea to create a new setting in pocketmine.yml for experimental gameplay for both that line and https://github.com/Spajker7/PocketMine-MP/blob/6c567e7c14d221026938c52b4bb2f96846b6aec2/src/pocketmine/Player.php#L2091

but it seems like a huge pain in the ass right now. It'd probably also require making a better system for game rules. 

Lets see what Dylan thinks.

Reply:
Honestly, who cares. The client is always a broken piece of shit anyway, so adding switches for enabling some "experimental" features that 99% of users won't use is just going to get in the way.